### ISSUE TITLE
Documentation for Science users

### ISSUE THREAD

**Issue by sfiruch** [i4Ds/Karabo-Pipeline#14]
We should provide an introduction & overview for science users.

- [ ] Overview
- [ ] Tutorials
- [x] Examples
- [ ] FAQs

**Comment by cvoegele** [comment]
I think this should also include a generated API overview with examples built right into that. Or not? I think we could use a CI trigger to then push the new documentation to GitHub pages automatically.

**Comment by Lukas113** [comment]
I think we already cover all the points of this issue once #583 is completed!

**Comment by sfiruch** [comment]
Don't think so. #583 is about PRs and docstrings, not about science.

@anawas Can we make your tutorial available online for science users?

**Comment by anawas** [comment]
I agree with @sfiruch. We need to update and enhance the documentation for science users. This should be done as demo code in Jupyter notebook and as commented code samples in online documentation.
The following topics should be covered:

- [ ] How to add noise
- [ ] How to include user defined beams
- [ ] Source detection feature
- [ ] Working with IVOA ObsCore metadata

**Comment by mpluess** [comment]
Source detection should be covered here: https://i4ds.github.io/Karabo-Pipeline/examples/examples.html#source-detection
Metadata is somewhat covered here, but might need additions / clarifications: https://i4ds.github.io/Karabo-Pipeline/examples/examples.html#srcnet
